mlir/test/CAPI/mixr_full.c

```diff
@@ -194,7 +194,7 @@ static bool constructAndTraverseIr(MlirContext ctx) {
       mlirRockTuningSpaceCreate(module, RocmlirTuningParamSetKindFull);
   printf("Got tuning space,\n");
   unsigned fNum = mlirRockTuningGetNumParams(tuningSpace);
-  // CHECK: full set = 357
+  // CHECK: full set = 461
```

Comment:
There is a problem here. Due to some changes in GridwiseGemmParams.cpp not included in release6.1, this test case failed because it generated "full set = 475" here. Should I fix it in release6.1?

Reply:
Oh just realized that this is a backporting PR. 

If you want to backport it, yah, this will have to be fixed to merge